querydsl-sql-spatial/pom.xml

```diff
@@ -1,4 +1,4 @@
-<?xml version="1.0" encoding="UTF-8"?>
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Did you find out yet what causes these changes?
Is it required by spec to end the prelude in CRLF?
Or are they now LF?